src/lib/app_registry.tsx

```diff
@@ -91,7 +91,16 @@ class WorksForYou extends RootContainer {
   }
 }
 
+class MyProfile extends RootContainer {
```

Comment:
In that case, yeah, I think it'd be a good change to make now while it's still fairly easy :D